### PR TITLE
Feature/species page changes

### PR DESCRIPTION
**Pull request by dfrico**
Features requested changes:
- Vertical spacing in the countries selection view.
- Faster animation in same view.
- Dot, line, and legend tweaks in species chart.

Also:
- Importer script from a data.json to graphCMS.

### CODE REVIEW

**Comment by aabdaab** [comment]
could you rebase with develop cause I rewrote the `countries` component to allow for countries abstraction and now there are some conflicts 🙏

File: src/pages/home/components/countries/styles.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    .shape__no-hover {
+      opacity: 1;
+      transition: all 0.05s $ease-in-out-sine;
+      transform: scale3d(1, 1, 1);
+      transform-origin: left;
+    }
+  }
+}
```
Comment (aabdaab) [inline src/pages/home/components/countries/styles.scss]: you can remove that file as it is not used anywhere (that's a leftover from merge)

File: src/pages/species/components/species-list/styles.scss [hunk 2]
```diff
@@ -18,6 +18,7 @@
     &.__active{
       font-size: 30px;
       height: 70px;
+      background-color: #313131;
```
Comment (aabdaab) [inline src/pages/species/components/species-list/styles.scss]: move it to settings color variables? you can use http://chir.ag/projects/name-that-color this handy page to name colors

**Comment by sorodrigo** [comment]
The species list box, should be inside another box with fixed size, so that the scroll doesn't move the page and the image is always fixed. Otherwise you can end up in this situation:

Screen Shot 2019-10-17 at 11 10 26

**Pull request merged**